neurodsp/sim/combined.py

```diff
@@ -127,38 +127,35 @@ def sim_peak_oscillation(sig_ap, fs, freq, bw, height):
     """
 
     sig_len = len(sig_ap)
-    times = create_times(sig_len/fs, fs)
+    times = create_times(sig_len / fs, fs)
 
-    # Construct the aperiodic component and compute its Fourier transform. Only use the
-    # first half of the frequencies from the Fourier transform since the signal is real.
-    sig_ap_hat = np.fft.fft(sig_ap)[0:(sig_len//2+1)]
+    # Construct the aperiodic component and compute its Fourier transform
+    # Only use the first half of the frequencies from the FFT since the signal is real
+    sig_ap_hat = np.fft.fft(sig_ap)[0:(sig_len // 2 + 1)]
 
     # Create the range of frequencies that appear in the power spectrum since these
     # will be the frequencies in the cosines we sum below
-    freqs = np.linspace(0, fs/2, num=sig_len//2 + 1, endpoint=True)
+    freqs = np.linspace(0, fs / 2, num=sig_len // 2 + 1, endpoint=True)
 
     # Construct the array of relative heights above the aperiodic power spectrum
-    rel_heights = np.array([height * np.exp(-(lin_freq - freq)**2/(2*bw**2)) for lin_freq in freqs])
+    rel_heights = np.array([height * np.exp(-(lin_freq - freq) ** 2 / (2 * bw ** 2)) \
+        for lin_freq in freqs])
 
-    # Build an array of the sum of squares of the cosines as they appear in the calculation of the
-    # amplitudes
-    cosine_norms = np.array(
-        [norm(np.cos(2*np.pi*lin_freq*times), 2)**2 for lin_freq in freqs]
-    )
+    # Build an array of the sum of squares of the cosines to use in the amplitude calculation
+    cosine_norms = np.array([norm(np.cos(2 * np.pi * lin_freq * times), 2) ** 2 \
+        for lin_freq in freqs])
 
     # Build an array of the amplitude coefficients
-    cosine_coeffs = np.array(
-        [(-np.real(sig_ap_hat[ell]) + np.sqrt(np.real(sig_ap_hat[ell])**2 + \
-         (10**rel_heights[ell] - 1)*np.abs(sig_ap_hat[ell])**2))/cosine_norms[ell]
-         for ell in range(cosine_norms.shape[0])]
-    )
+    cosine_coeffs = np.array([\
+        (-np.real(sig_ap_hat[ell]) + np.sqrt(np.real(sig_ap_hat[ell]) ** 2 + \
+        (10 ** rel_heights[ell] - 1) * np.abs(sig_ap_hat[ell]) ** 2)) / cosine_norms[ell] \
+        for ell in range(cosine_norms.shape[0])])
 
     # Add cosines with the respective coefficients and with a random phase shift for each one
-    sig_periodic = np.sum(
-        np.array([cosine_coeffs[ell]*np.cos(2*np.pi*freqs[ell]*times + 2*np.pi*np.random.rand())
-                  for ell in range(cosine_norms.shape[0])]),
-        axis=0
-    )
```

Comment:
Throughout this func, I prefer the old format a bit more. It feels slightly more readable but nbd either way since it's equivalent.